extensions/system-information/src/index.jsx

```diff
@@ -0,0 +1,106 @@
+import { useState, useEffect } from "react";
+import { Icon, List } from "@raycast/api";
+import { macOSVersion } from "macos-version";
+import macosRelease from "macos-release";
+import os from "os";
+import si from "systeminformation";
+import { exec } from "child_process";
+async function calculateDiskStorage() {
+  const disks = await si.fsSize();
+
+  const diskObj = disks[0];
+  const totalSize = (diskObj.size / (1024 * 1024 * 1024)).toFixed(2);
+  const totalAvailableStorage = (diskObj.available / (1024 * 1024 * 1024)).toFixed(2);
+
+  return `${totalAvailableStorage} GB available of ${totalSize} GB`;
+}
+
+export default function Command() {
+  const [storageInfo, setStorageInfo] = useState("");
+  const [serialNumber, setSerialNumber] = useState("");
+
+  useEffect(() => {
+    calculateDiskStorage().then((size) => {
+      setStorageInfo(size);
+    });
+
+    exec("/usr/sbin/system_profiler SPHardwareDataType", (error, stdout, stderr) => {
+      if (error) {
+        console.error(`Error: ${error.message}`);
+        return;
+      }
+      if (stderr) {
+        console.error(`stderr: ${stderr}`);
+        return;
+      }
+
+      const serialNumberMatch = stdout.match(/Serial Number \(system\): (.+)/);
+      const serialNumber = serialNumberMatch ? serialNumberMatch[1] : null;
+
+      setSerialNumber(serialNumber);
+    });
+  }, []);
+
+  const releaseImage = () => {
```

Comment:
You forgot about Sonoma, and it would be cool if images were added to assets instead.